setup.py

```diff
@@ -22,4 +22,5 @@
         "console_scripts": ["cohortextractor=datalab_cohorts.cohortextractor:main"]
     },
     include_package_data=True,
+    classifiers=["License :: OSI Approved :: GNU General Public License v3 (GPLv3)"],
```

Comment:
Not disagreeing with this choice, but was there any discussion around this? In the past we've opted for MIT for Datalab stuff on the grounds of maximising permissiveness, but obviously the context is quite different here.